### PR TITLE
Pytest

### PR DESCRIPTION
**Pull request by kajuberdut**
Travis branch for #6 and the pytest branch for #4. I would have liked to keep these separate but apparently I'll need to find a GUI for git because it is opaque to me.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (gp-tmcfadden) [inline .vscode/settings.json]: Definitely remove this file from src code control.  Please add `.vscode` to `.gitignore` while you're at it.

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,17 @@
+help:
```
Comment (gp-tmcfadden) [inline Makefile:1]: `Makefile`?  That's old school!

File: Pipfile.lock [hunk 3]
```diff
@@ -18,11 +18,11 @@
     "default": {
         "django": {
             "hashes": [
-                "sha256:7f246078d5a546f63c28fc03ce71f4d7a23677ce42109219c24c9ffb28416137",
-                "sha256:ea50d85709708621d956187c6b61d9f9ce155007b496dd914fdb35db8d790aec"
+                "sha256:04f2e423f2e60943c02bd2959174b844f7d1bcd19eabb7f8e4282999958021fd",
+                "sha256:e1cc1cd6b658aa4e052f5f2b148bfda08091d7c3558529708342e37e4e33f72c"
             ],
```
Comment (gp-tmcfadden) [inline Pipfile.lock:23]: This is why we need to change `Pipfile` to lock down do a certain version of Django (IMO)

**Review by gp-tmcfadden — CHANGES_REQUESTED** [review]
Looks fine once you fix `.vscode`.

**Review by mtik00 — APPROVED** [review]
LGTM

**Pull request merged**